### PR TITLE
002 create project table

### PR DESCRIPTION
**Pull request by hannahmorgan24**

### CODE REVIEW

**Comment by hannahmorgan24** [comment]
All checks passed!

File: src/main/java/org/example/daos/ProjectDao.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class ProjectDao {
+
+
+    public int createProject(final ProjectRequest project) throws SQLException {
+
+        Connection c = DatabaseConnector.getConnection();
```
Comment (bbell-kainos) [inline src/main/java/org/example/daos/ProjectDao.java:16]: Dont forget to close this connection (maybe use try/finally?)

File: src/main/java/org/example/models/Project.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+public class Project {
+
+    private String projectName;
+    private double projectValue;
+    private int clientID;
+
+    @JsonCreator
+    public Project(
+            @JsonProperty("projectName1") final String projectName1,
```
Comment (bbell-kainos) [inline src/main/java/org/example/models/Project.java:14]: Any reason why these are named 'somethingSomething1'

**Review by bbell-kainos — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/models/Project.java (hunk 2 above)
Comment (hannahmorgan24) [reply src/main/java/org/example/models/Project.java:14]: some of the CheckStyle errors were telling me they were duplicate names and to rename them?

**Review by bbell-kainos — APPROVED** [review]

**Pull request merged**